sambamba.wdl

```diff
@@ -38,10 +81,10 @@ task Markdup {
         # 2 threads reduces wall clock time by more than 40%.
         Int threads = 2
         # According to the manual sambamba markdup uses the sortbufferSize + 2 times the ioBuffer size.
-        # Added 1024 mb as a margin of safety. Real life use with this setting uses 2.7 GiB.
-        Int memoryMb = 1024 + sortBufferSize + 2 * ioBufferSize
+        # Added 8192 mb as a margin of safety. Real life use with this setting uses 2.7 GiB.
```

Comment:
Is the rest of the comment still true? That is quite a lot of memory to add for safety. That is overprosioning with quite a lot of GB. Is there another buffer that the application uses that is not been taken into account?

Reply:
I'm not sure why, but the process would occasionally need a lot more memory.

Reply:
Probably when mates are far apart. This is more common in RNA data.